profile.php

```diff
@@ -336,6 +347,23 @@ function userProfile() {
 		<label style='width:9.2em;'>Speed:</label>" . sprintf("%.2f", $player->speed) . "<br />
 		<label style='width:9.2em;'>Intelligence:</label>" . sprintf("%.2f", $player->intelligence) . "<br />
 		<label style='width:9.2em;'>Willpower:</label>" . sprintf("%.2f", $player->willpower) . "<br />
+		<br />
+		<b>Missions Completed:</b><br />
+		&nbsp;&nbsp;<label style='width:5em;'>" . Mission::$rank_names[Mission::RANK_D] . ":</label>
+		    " . (isset($player->missions_completed[Mission::RANK_D]) ? $player->missions_completed[MIssion::RANK_D] : '0') . "

```

Comment:
```suggestion
		    " . (isset($player->missions_completed[Mission::RANK_D]) ? $player->missions_completed[Mission::RANK_D] : '0') . "

```
And the rest of these